blueman/main/DBusProxies.py

```diff
@@ -8,7 +10,8 @@ class DBusProxyFailed(Exception):
 
 
 class ProxyBase(Gio.DBusProxy, metaclass=SingletonGObjectMeta):
-    def __init__(self, name, interface_name, object_path='/', systembus=False, flags=0, *args, **kwargs):
+    def __init__(self, name: str, interface_name: str, object_path: str = "/", systembus: bool = False,
+                 flags: Gio.DBusProxyFlags = 0, *args: Any, **kwargs: Any) -> None:
```

Comment:
Feel free to drop the *args **kwargs as I doubt we actually make use of them. If we do add a new keyword and annotate it properly :-)